GameServer.h

```diff
@@ -122,9 +122,7 @@ class GameServer
 
 	SOCKET hostSocket;
 	SOCKET secondarySocket;
-	GameInfo* gameInfo;
-	unsigned int numGames;
-	unsigned int maxNumGames;
+	std::vector<GameInfo> gameInfos;

```

Comment:
I was thinking about renaming the type GameInfo to GameSession. The plural would then be gameSessions which I thought sounded better. 

It would cause a rename of the class CreateGameInfo as well. Maybe GameSessionCreation. This is an inherited type though from packet.h and I wasn't sure it could be renamed.

Reply:
It sounds like you are ok if we make the GameSession & CreateGameSession rename now and  defer structures in the packet header until post repo-merge.